integration/tests/access/access_api_test.go

```diff
@@ -35,7 +35,7 @@ func (suite *AccessSuite) TearDownTest() {
 		defer suite.cancel()
 	}
 	if suite.net != nil {
-		suite.net.Remove()
+		//suite.net.Remove()
```

Comment:
Why is this commented out?

Reply:
PR is a WIP, because running the integration tests locally is resource intensive I am using CI to let me know what test are failing.